packages/client/README.md

```diff
@@ -132,6 +132,30 @@ ethereumjs --mine --unlock=[ADDRESS1],[ADDRESS2],...
 
 Note that this feature is in `beta` and shouldn't be used with accounts holding a substantial amount of `Ether` on mainnet (or other valuable assets) for security reasons.
 
+### Custom network for development
+
+The client provides a quick way to get a local instance of the block chain up and running using the `--dev` command. This will start up a private PoA clique
```

Comment:
```suggestion
The client provides a quick way to get a local instance of a blockchain up and running using the `--dev` command. This will start up a private PoA clique
```